app/src/main/java/com/sunasterisk/appchat/db/firebase/RemoteConstant.kt

```diff
@@ -4,4 +4,6 @@ object RemoteConstant {
 
     const val EMAIL_NOT_VERIFIED = "email_not_verified"
     const val MSG_EMAIL_NOT_VERIFIED = "Email is not verified"
+    const val COLLECTION_USER = "Users"
+    const val MSG_ACCOUNT_HAS_BEEN_EXISTED = "User has already registered!"
```

Comment:
message này để ra strings.xml chứ em?

Reply:
hình như ko cần anh ạ, vì Google Sign In nếu user đã tồn tại hay chưa thì cũng đều trả về Success<User> chứ anh nhỉ, chỉ nếu chưa thì lưu thêm vào firestore thôi